quanttrader/strategy/strategy_base.py

```diff
@@ -113,13 +113,11 @@ def adjust_position(self, sym, size_from, size_to, timestamp=None):
         o.full_symbol = sym
         o.order_type = OrderType.MARKET
         o.order_size = size_to - size_from
-        o.source = self.id  # identify source
-        if timestamp is None:
-            o.create_time = datetime.now().strftime('%Y-%m-%d %H:%M:%S.%f')
-        else:
+        if timestamp is not None:
             o.create_time = timestamp
-        if (self.active):
-            self.strategy_manager.place_order(o)
+
```

Comment:
On the trade window there is a strategy control window to activate/pause/stop a strategy. This is managed by StrategyManager. So if this strategy is inactive, then StrategyManager will not place the order.

Reply:
This is also done place_order (). adjust_position () will just call place_order ()